core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -25,42 +25,48 @@
   {{dependencies_html}}
 {% endblock header_js %}
 
+{% block navbar_breadcrumb %}
+  <collection-editor-navbar-breadcrumb-directive>
+  </collection-editor-navbar-breadcrumb-directive>
+{% endblock navbar_breadcrumb %}
+
+{% block local_top_nav_options %}
+  <collection-editor-navbar-directive>
+  </collection-editor-navbar-directive>
+{% endblock local_top_nav_options %}
+
 {% block content %}
   <div ng-controller="CollectionEditor">
-    <div ng-if="collection">
-      <!-- Collection details -->
-      <collection-details-editor-directive collection="collection">
-      </collection-details-editor-directive>
-      <a ng-href="/collection/<[collection.getId()]>" target="_blank">Go to Collection</a>
-
-      <!-- List of acquired and pre-requisite skills -->
-      <br>
-      <br>
-      <strong>Skills</strong>
-      <collection-skill-list-directive collection-skill-list="collectionSkillList">
-      </collection-skill-list-directive>
-
-      <!-- List of explorations -->
-      <br>
-      <strong>Explorations</strong>
-      <ul>
-        <li ng-repeat="node in collection.getBindableCollectionNodes()">
-          <collection-node-editor-directive collection="collection" collection-node="node" update-skill-list="updateSkillList()">
-          </collection-node-editor-directive>
-        </li>
-      </ul>
-
-      <!-- Collection changes to be commited -->
-      <h3><span ng-if="getChangeListCount() > 0"> <em>(<[getChangeListCount()]> changes pending)</em></h3>
-      <br>
-      <input ng-model="commitMessage" type="text" placeholder="Commit Message"><button ng-click="saveCollection(commitMessage)">Save Collection</button>
-      <button ng-click="publishCollection()" ng-if="!isPublic">Publish Collection</button>
+    <div ng-if="getTabStatuses().active === 'main'">
+      {% include 'collection_editor/collection_main.html' %}
+    </div>
+
+    <div ng-if="getTabStatuses().active === 'preview'">
+      {% include 'collection_editor/collection_preview.html' %}
+    </div>
+
+    <div ng-show="getTabStatuses().active === 'settings'">
+      {% include 'collection_editor/collection_settings.html' %}
+    </div>
+
+    <div ng-show="getTabStatuses().active === 'history'">
+      {% include 'collection_editor/collection_history.html' %}
+    </div>
+
+    <div ng-show="getTabStatuses().active === 'feedback'">
+      {% include 'collection_editor/collection_feedback.html' %}
+    </div>
+
+    <div ng-show="getTabStatuses().active === 'stats'">
+      {% include 'collection_editor/collection_statistics.html' %}
     </div>
   </div>
 
   {% include 'collection_editor/collection_details_editor_directive.html' %}
   {% include 'collection_editor/collection_node_editor_directive.html' %}
   {% include 'collection_editor/collection_skill_list_directive.html' %}
+  {% include 'collection_editor/collection_editor_navbar_breadcrumb_directive.html' %}
```

Comment:
Nit: probably better to keep this in alphabetical order unless they need to be placed out of order.